plugins/plugin-maven/maven-server/maven-server-impl/src/main/java/org/eclipse/che/maven/server/MavenServerImpl.java

```diff
@@ -425,8 +425,9 @@ public MavenArtifact resolveArtifact(MavenArtifactKey artifactKey, List<MavenRem
                                                                              new ArtifactRequest(RepositoryUtils.toArtifact(artifact),
                                                                                                  remoteRepositories, null));
         return MavenModelUtil.convertArtifact(RepositoryUtils.toArtifact(artifactResult.getArtifact()), localRepository);
-        } catch (ArtifactResolutionException e) {
-            MavenServerContext.getLogger().info(e);
+        } catch (ArtifactResolutionException ignored) {
+            //we need ignore exception, it's some times has class that client doesn't has
```

Comment:
All sdtout/stderr of the maven sever is logged by client, but 'ArtifactResolutionException' is normal when maven can't find or download artifact, so I not sure that this exception need to be logged.